lib/python/vdm/server/voltdbserver.py

```diff
@@ -289,6 +292,40 @@ def start_local_server(self, sid, recover=False):
         else:
             return 1
 
+    def get_volt_deploy_root_folder(self, server, sid):
+        path = ''
```

Comment:
I think this code can be simplified substantially. server id is unique across databases, correct? So, we can simply name the directory server_<server_id>. Then you can avoid the  code to loop through all the directories and all that.